themes/bootstrap3/templates/RecordDriver/SolrOverdrive/status-full.phtml

```diff
@@ -33,14 +33,15 @@ foreach($ids as $id):
     } else {
         $current['full_status']
             = "<p><div class='copies'><strong>Total Copies</strong> "
-            . $avail->copiesOwned . "</div>
+            . $avail->copiesOwned . "</div>
                         <div class='avail'><strong>Available</strong> "
-            . $avail->copiesAvailable . "</div>
+            . $avail->copiesAvailable . "</div>
                         <div class='holds'><strong>Holds</strong> "
             . $avail->numberOfHolds . "</div></p>";
     }
   $data[] = $current;
-endforeach;
+}
+$jsonResult->data = new \stdClass;
```

Comment:
`$jsonResult` could be instantiated here as it's not needed before. But do we need stdClass or could $jsonResult be just an associative array?